src/dnaio/_core.pyx

```diff
@@ -1,16 +1,20 @@
 # cython: language_level=3, emit_code_comments=False
 
+from cpython.bytes cimport PyBytes_FromStringAndSize, PyBytes_AS_STRING, PyBytes_Check, PyBytes_GET_SIZE
+from cpython.unicode cimport PyUnicode_DecodeLatin1, PyUnicode_Check
```

Comment:
There’s already a `from cpython.unicode cimport ...` line below, can you please merge them?